spec/services/voucher_adjustments_service_spec.rb

```diff
@@ -342,6 +348,12 @@
         included_tax: 0,
         amount: 0.5
       )
+      AdjustmentMetadata.create(
+        adjustment: tax_adjustment,
+        enterprise: tax_adjustment.originator.enterprise,
+        fee_name: "Tax",
+        fee_type: "Voucher"
+      )
```

Comment:
👀 This is now quite a bit of manual set up duplicated from the application code..

Reply:
I fix that, thanks for prompting me to re visit this :pray: